l3kernel/l3debug.dtx

```diff
@@ -787,6 +787,7 @@
       \prop_set_eq:NN
       \prop_set_from_keyval:Nn
       \seq_set_eq:NN
+      \seq_set_from_clist:Nn
```

Comment:
Wondering why this change along won't fail the checks.

Reply:
Oh, the problem of setting `\l_graphics_search_ext_seq` before declaration only appears in non-LaTeX formats:

https://github.com/latex3/latex3/blob/05c22772e096b48aac4addfbb8b3b93a70e803ee/l3backend/l3backend-graphics.dtx#L66-L76

One of the backends
https://github.com/latex3/latex3/blob/05c22772e096b48aac4addfbb8b3b93a70e803ee/l3backend/l3backend-graphics.dtx#L83-L89

Reply:
And, since generic loader (`expl3-generic.tex`) loads both the expl3 kernel (`expl3-code.tex`) and backend driver, there's currently no way to turn on debugging between loading kernel and driver.

Currently (before this PR) with little tweak `l3graphics.sty` can be loaded in plain TeX:
```tex
\input expl3-generic.tex\relax

\ExplSyntaxOn
\cs_new_eq:NN \ProvidesExplPackage \use_none:nnnn
\input l3graphics.sty\relax
\ExplSyntaxOff

\end
```
Compile it and you'll get an error
```
! LaTeX Error: Control sequence \l_graphics_search_ext_seq already defined.
```

Thus the question becomes: does `l3graphics` aim to be used in non-LaTeX formats (in the future), or we live with the small flaw that a variable would be set before declaration when `l3graphics` is used in unsupported formats? @josephwright

Reply:
Now `\l_graphics_search_ext_seq` is declared in `l3graphics` in LaTeX format, and in backend drivers otherwise.